init.sls

```diff
@@ -65,14 +65,17 @@ chown_pgdata:
       autovacuum_vacuum_threshold: 50
       autovacuum_vacuum_scale_factor: 0.2
       log_autovacuum_min_duration: '500ms'
+      logging_collector: false
```

Comment:
I think we should leave `logging_collector` out here, so there's no tampering with the defaults. The [docs](https://www.postgresql.org/docs/11/runtime-config-logging.html) do not mention a default, unless I was missing something here. 
Furthermore, this setting has a lot of implications (see below). 

I think the better idea is to not set a default here, and only set the respective settings when the pillar file is set (analogue to the other settings below).

Reply:
Fine for me. However I think the example pillar should include it then, in addition to all the other possible configurations. Otherwise your NOTE at the beginning of the config section in pillar.example is not accurate anymore.
What do you think?

Reply:
Agree. All supported options should be documented in the `pillar.example`.

Reply:
Ok, see a403b56787fa1834a0742622d19cc0db2c8b6381 . I am not quite sure about the defaults though. All (unset) would be an option as well, to hint toward leaving them out entirely.